submissions/Roka20012/a-tiny-JS-world/index.js

```diff
@@ -42,8 +42,8 @@ const man = {
   saying: 'Hello Jenny!'
 };
 
-let creatures = [dog, cat, woman, man];
-
-creatures.forEach(el => {
-  print(`Hi! I am ${el.species}; My name is <strong>${el.name}</strong>; My gender is <em>${el.gender}</em>;I have ${el.legs} legs;I have ${el.hands} hands;I want to say ${el.saying};`);
+[dog, cat, woman, man].forEach(el => {
+  print(`Hi! I am ${el.species}; My name is <strong>${el.name}</strong>;
+    My gender is <em>${el.gender}</em>;I have ${el.legs} legs;
+    I have ${el.hands} hands;I want to say ${el.saying};`);
```

Comment:
Actually this nicer, but I've meant just space after last semicolons :D

Reply:
`;I have` `hands;I`

Reply:
Fix it?)Maybe :)